icclim/ecad_functions.py

```diff
@@ -653,6 +653,12 @@ def _can_run_bootstrap(config: IndexConfig, percentile_period) -> bool:
         config.cf_variables[0].in_base_da is not config.cf_variables[0].da
         and len(overlapping_years) > 1
     )
+    if run_bootstrap:
+        in_base = config.cf_variables[0].in_base_da
+        chunking = {d: "auto" for d in in_base.dims}
```

Comment:
ugly, do dict() wrapping

Reply:
Meh, `dict.fromkey` causes types issues as DataArray.chunk expects a Mapping not a dict.